main.py

```diff
@@ -1,7 +1,40 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
+
+
+genres = ["Western", "Action", "Dramma"]
+actors = [
+    ["George", "Klooney"],
+    ["Kianu", "Reaves"],
+    ["Scarlett", "Keegan"],
+    ["Will", "Smith"],
+    ["Jaden", "Smith"],
+    ["Scarlett", "Johansson"]
+]
 
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(
+            name=genre,
+        )
+
+    for actor in actors:
+        first_name, last_name = actor
```

Comment:
Perform unpacking during the loop, there is no need for this line
```suggestion
    for  first_name, last_name in actors:
```